addons/knowledge/models/knowledge_article.py

```diff
@@ -1319,6 +1312,21 @@ def get_valid_parent_options(self, term=""):
             fields=['id', 'icon', 'name'],
         )
 
+    def _get_first_accessible_article(self):
```

Comment:
Ordering: would define it after get_invite_url (because above and under, we have methods about parents / children, let us have methods by category)